printing/ppvernac.ml

```diff
@@ -538,14 +538,6 @@ open Decl_kinds
   let rec pr_vernac_expr v =
```

Comment:
Good catch. Addressed below. I also added two occurrences of the `nonrec` keyword. It has been part of the OCaml language since version 4.02.2 (according to the [OCaml manual](http://caml.inria.fr/pub/docs/manual-ocaml/typedecl.html)); is that OK?

Reply:
It's fine, there are already some in a few places.

Reply:
That's good @vbgl , minimal OCaml for Coq is 4.02.3, note however that if you would like to use some newer feature in code, you can open an issue as a reminder and tag it `needs: newer ocaml`.